sos/plugins/memory.py

```diff
@@ -37,7 +37,7 @@ def setup(self):
         self.add_cmd_output("free", root_symlink="free")
         self.add_cmd_output([
             "free -m",
-            "swapon --bytes --show"
+            "swapon --summary --verbose"
```

Comment:
Holy crap.. that package is from mid-2011!!

```
commit 60a5833b64bd185de4746331a68352d902051ce8
Author: Karel Zak <kzak@redhat.com>
Date:   Mon Aug 29 13:03:13 2011 +0200

    build-sys: release++ v2.20
    
    Signed-off-by: Karel Zak <kzak@redhat.com>

```

For a distro released in 2014 that is pretty shocking.

I'm happy if Ubuntu wants to subclass and special-case this, but that is *so* old I don't think we want to take any special steps to cater to it in the main plugin class.